webapp/commonComponents/windowConfirm.js

```diff
@@ -0,0 +1,43 @@
+import './windowConfirm.scss'
```

Comment:
I would call this module ConfirmDialog...

Reply:
ok